app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -65,5 +55,16 @@ def textual_product_name
       name = @record.operating_system.try(:product_name)
       {:label => "Product Name", :value => name} if name
     end
+
+    def textual_guest_applications
```

Comment:
@dkorn I have the feeling this could be simplified (for sure not a copy-paste). It should probably be:

    textual_link(@record.guest_applications)

cc @moolitayer @zeari